CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.1.98 - Minor changes in tailwind.config.js
```

Comment:
@mcibique when you get a chance I'd like to know if you see any drawbacks to these changes I made to the tailwind config.